backend/src/routes/user.ts

```diff
@@ -112,4 +125,160 @@ router.post("/mentor", [validateMentor], async (req: Request, res: Response) =>
   }
 });
 
+/**
+ * This is a get route for a mentee. Note that the response is dependant on
+ * the person calling the method
+ *
+ * Mentee calling: gain all info about the mentee
+ *
+ * Mentor calling: mentee name, image, grade, about, career interests, topics of interest
+ */
+router.get("/mentee/:userId", [verifyAuthToken, upload], async (req: Request, res: Response) => {
```

Comment:
Why do we need upload for a GET request? Isn't `upload` just for the case where we upload an image into the server?

Reply:
It was there to test uploading an image to MongoDB but it isn't needed for the get route.